src/snmpd/patch-5.7.3+dfsg/series

```diff
@@ -4,4 +4,4 @@
 0004-Disable-SNMPv1.patch
 0005-Port-OpenSSL-1.1.0-with-support-for-1.0.2.patch
 0006-From-Jiri-Cervenka-snmpd-Fixed-agentx-crashing-and-or-freezing-on-timeout.patch
-0007-Linux-VRF-5.7.3-Support.patch
```

Comment:
do not remove this.

Reply:
I will work with the feature owner to make it compatible with ipv6 code. They are enabled by conditional compilation.

Reply:
this is owned by dell.